Ktisis/Interface/ConfigGui.cs

```diff
@@ -3,7 +3,15 @@
 
 using ImGuiNET;
 
+using Dalamud.Interface.Components;
```

Comment:
Double `using` statements here and in a couple of other places. Looks like an issue with conflict resolution. (If you need help, ask.)